Kudu.Contracts/SiteExtensions/ISiteExtensionManager.cs

```diff
@@ -17,7 +17,7 @@ public interface ISiteExtensionManager
         /// <summary>
         /// Install or update a site extension
         /// </summary>
-        Task<SiteExtensionInfo> InstallExtension(string id, string version, string feedUrl, SiteExtensionInfo.SiteExtensionType type, ITracer tracer);
+        Task<SiteExtensionInfo> InstallExtension(string id, string version, string feedUrl, SiteExtensionInfo.SiteExtensionType type, ITracer tracer, string installationArgs);
```

Comment:
These are purely Kudu internal interface that no one else consumes, so no concerns with changing signature.

Agreed that a comment with some sample values will help clarify.